### ISSUE TITLE
confused by devDependencies status

### ISSUE THREAD

**Issue by getify** [alanshaw/david-www#373]
I'm confused by this... it clearly shows one of my dev dependencies as out of date, but the badge still says "up to date". Am I missing something?

![screen shot 2017-03-28 at 5 08 25 pm](https://cloud.githubusercontent.com/assets/150330/24429865/809a1090-13d9-11e7-87ff-2aff528e916a.png)

**Comment by dolanmiu** [comment]
Your issue is basically this issue:

https://github.com/alanshaw/david-www/issues/367

**Comment by getify** [comment]
@dolanmiu seems like my symptoms are opposite of those in that thread. could be same underlying cause, or could be a different problem.

**Comment by dolanmiu** [comment]
I would guess it's most likely the same underlying issue. It's essentially the status being de-synced.

Maybe it's lagging behind for some reason

**Comment by alanshaw** [comment]
Badges may lag behind what you see on the status page slightly. Badge update requests are queued and serviced separately to page requests for the same information. If you wait for a few minutes and refresh the page you should see an updated badge (exact time depends on how busy the queue is).

**Comment by getify** [comment]
> may lag behind what you see on the status page slightly... If you wait for a few minutes and refresh the page you should see an updated badge

I can certainly understand a few minutes or even a few hours (up to a full day!) lag. When I filed the OP, I was **weeks** past when the package in question (quint) had gone out of date. It was updated to 2.2.0 (and then 2.2.1) on Mar 19, but this OP was on Mar 28 and the badge was still lagging. That clearly seems like more than just normal propagation.

BTW, I cleared cache and tried in several browsers and devices at the time, so it was a cache on the service side that was at least 9 days outdated.
